ui/apps/platform/src/Containers/Vulnerabilities/VulnerablityReporting/VulnReports/OnDemandReportsTab.tsx

```diff
@@ -1,10 +1,48 @@
-import React from 'react';
+import React, { useCallback } from 'react';
+import { Card, CardBody, PageSection, Text } from '@patternfly/react-core';
+
+import { getTableUIState } from 'utils/getTableUIState';
+import useRestQuery from 'hooks/useRestQuery';
+import { fetchOnDemandReportHistory } from 'services/ReportsService';
 import PageTitle from 'Components/PageTitle';
+import OnDemandReportsTable from './OnDemandReportsTable';
 
 function OnDemandReportsTab() {
+    // @TODO: Pass query, pagination, sorting to API call
+    const fetchOnDemandReportsHistoryCallback = useCallback(() => fetchOnDemandReportHistory(), []);
+    const { data, isLoading, error } = useRestQuery(fetchOnDemandReportsHistoryCallback);
+
+    // @TODO: Add polling
+
+    const tableState = getTableUIState({
+        isLoading,
+        data,
+        error,
+        searchFilter: {},
+        isPolling: false,
+    });
+
     return (
         <>
             <PageTitle title="Vulnerability reporting - On-demand reports" />
+            <PageSection variant="light">
+                <Text>
+                    Check job status and download on-demand reports in CSV format. Requests are
+                    purged according to retention settings.
+                </Text>
+            </PageSection>
+            <PageSection>
+                <Card>
+                    <CardBody className="pf-v5-u-p-0">
```

Comment:
Possibly for a discussion outside of this PR since I know we do this elsewhere, but do we need the Card wrapper here? It doesn't seem like wrapping the table in a `<Card>` fits the PatternFly component's use case. We can remove it and the extra `className` prop and the only visible difference is that the faint box shadow is removed.

Reply:
That's a good point. I thought about it too, but decided to keep it as we do in other sections. To be honest, we probably don't need it. If we're looking at a dashboard, it makes more sense. Since the page/section is dedicated to showing a table, maybe it doesn't make as much sense.